<?xml version="1.0" encoding="utf-8"?>
<report
	progress="">
	<activity
		result="Failed"
		duration="11.32s"
		durationms="11319"
		type="root"
		rid="ace63c3cb25cd96"
		totalerrorcount="1"
		totalwarningcount="0"
		totalsuccesstestcasecount="0"
		totalfailedtestcasecount="0"
		totalblockedtestcasecount="0"
		childsuccesscount="0"
		childfailedcount="0"
		childblockedcount="0"
		totalsuccesscount="0"
		totalfailedcount="0"
		totalblockedcount="0"
		totalmaintenancemodecount="0"
		user="zeyne"
		rxversion="10.1.7.0"
		host="DESKTOP-E1ASG9D"
		osversion="Windows 10 64bit"
		runtimeversion="4.0.30319.42000"
		procarch="32bit"
		oslanguage="tr-TR"
		dtnlanguage="tr-TR"
		screenresolution="1920x1080"
		timestamp="28.04.2022 18:16:08"
		timestampiso="2022-04-28T15:16:08"
		endtime="28.04.2022 18:16:19"
		timeoutfactor="1"
		runid=""
		sutversion="">
		<detail>
			Ranorex Testscript
		</detail>
		<item
			timeRelativeToTestSuiteStartTime="00:11.311"
			timeRelativeToTestModuleStartTime="00:11.311"
			timeWallClock="18:16:19"
			time="00:11.311"
			level="Error"
			category="Module">
			<message>
				The test run has been aborted by the user.
			</message>
			<metainfo
				loglvl="Error">
			</metainfo>
		</item>
	</activity>
</report>
